Gemfile

```diff
@@ -1,4 +1,2 @@
 source 'https://rubygems.org'
 gemspec
-
-gem 'rubocop'
```

Comment:
FYI: This gem should be moved to `api_auth.gemspec` to `add_development_dependency`.